YarnSpinner/YarnSpinnerLexer.cs

```diff
@@ -52,9 +52,18 @@ public const int
 		LPAREN=64, RPAREN=65, COMMA=66, VAR_ID=67, BODY_NUMBER=68, FUNC_ID=69, 
 		COMMAND_UNKNOWN=70, ACTION=71, OPTION_SEPARATOR=72, OPTION_TEXT=73, OPTION_CLOSE=74, 
 		OPTION_LINK=75;
-	public const int
-		Title=1, Tags=2, HeaderText=3, Body=4, Text=5, Shortcuts=6, Command=7, 
-		Action=8, Option=9, OptionLink=10;
+
+    public const int Title = 1;
+    public const int Tags = 2;
+    public const int HeaderText = 3;
+    public const int Body = 4;
+    new public const int Text = 5;
```

Comment:
This constant hides a base member and it would probably be a bad idea to suppress that warning in general.  
Having the hiding marked explicitly here is in my opinion worth breaking up the formatting. (I'd personally break this up into an enum, but that's definitely out of scope here and I don't want to be prescriptive regarding style.)